frappe/public/js/frappe/desk.js

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
 // MIT License. See license.txt
 /* eslint-disable no-console */
+import Vue from 'vue/dist/vue.js';
```

Comment:
Vue should be required conditionally and should not exist in the default bundle

Reply:
Yeah, agreed. Did changes accordingly.